test/e2e/utils/cnpspecbuilder.go

```diff
@@ -27,6 +27,13 @@ type ClusterNetworkPolicySpecBuilder struct {
 	Name string
 }
 
+type ACNPRuleAppliedToSpec struct {
+	PodSelector         map[string]string
+	NSSelector          map[string]string
+	PodSelectorMatchExp *[]metav1.LabelSelectorRequirement
+	NSSelectorMatchExp  *[]metav1.LabelSelectorRequirement
```

Comment:
Not really introduced by this PR, but I don't recall why these have to be pointers, while `PodSelector` / `NSSelector` do not. Do you know the reason?

Reply:
I'm not sure... Maybe because Maps are already passed by reference?

Reply:
that may make sense for function parameters (except if we never mutate the slice), but not really for fields in a struct

Reply:
maybe you can punt this to the later commit you mentioned in https://github.com/vmware-tanzu/antrea/pull/1396/files#r539671256

Reply:
Opened #1647 for this